packages/core/src/libraries/sso-connector.ts

```diff
@@ -246,6 +251,48 @@ export const createSsoConnectorLibrary = (queries: Queries) => {
     }
   };
 
+  const refreshTokenSetSecret = async (
```

Comment:
nit: can add some comments, before reading the code, we do not know what this method is doing.

Reply:
comments added